python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -904,6 +904,22 @@ class NearestNeighbors(Base,
 
         return sparse_csr
 
+    @property
+    def metric(self, value):
```

Comment:
The property getter does not accept an argument to my knowledge.
```suggestion
    def metric(self):
```

Reply:
Oops, yes of course.